Cargo.toml

```diff
@@ -91,7 +91,7 @@ bundled-full = [
 ]
 
 [dependencies]
-time = "0.1.0"
+time = { version = "0.2", optional = true }
```

Comment:
Please, add this feature to [bundled-full](https://github.com/rusqlite/rusqlite/blob/master/Cargo.toml#L68) and [docs.rs](https://github.com/rusqlite/rusqlite/blob/master/Cargo.toml#L143).

Reply:
Done.